web/src/layout/Header/navbar/Menu/Settings/SendMeNotifications/FormNotifs/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { Checkbox, Button } from "@kleros/ui-components-library";
+import FormEmail from "./FormEmail";
+
+const FormContainer = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding-left: 32px;
+  padding-right: 32px;
+  padding-bottom: 32px;
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+  margin-top: 20px;
+`;
+
+const ButtonContainer = styled.div`
+  display: flex;
+  justify-content: end;
+  margin-top: 16px;
+`;
+
+const FormEmailContainer = styled.div`
+  position: relative;
+`;
+
+const EmailErrorContainer = styled.div`
+  position: absolute;
+  color: ${({ theme }) => theme.error};
+  font-size: 12px;
+  padding-top: 4px;
+  padding-left: 16px;
+`;
+
+const FormNotifs: React.FC = () => {
+  const [isWhenX, setIsWhenX] = useState<boolean>(false);
```

Comment:
turn this into a boolean array, then when you map over the checkboxes you can do:
```typescript
checkboxes.map(({ ... }, i) => <StyledCheckbox ..... onChange={(e) => setChecked(checked.splice(i, 1, e.target.checked) .... /> }
```